package.json

```diff
@@ -52,6 +52,7 @@
     "scripts/transforms"
   ],
   "dependencies": {
+    "@types/googlemaps": "3.39.6",
```

Comment:
Could this be added to `devDependencies` like the other `@types`?

Either way, we do not pin `@types` deps in this project.

Reply:
google maps is a dependency though, not dev dep, same with hogan.js, classnames, qs